Sources/TuistSupport/Logging/DetailedLogHandler.swift

```diff
@@ -73,3 +74,17 @@ func timestamp() -> String {
         }
     }
 }
+
+// MARK: - Protocol needed to suppress the warning because we don't have a valid `source` to pass to the `log` method
```

Comment:
Is this actually necessary? Looking at the source,

- The deprecated method currently being called sets the `source` to `Logger.currentModule(filePath: file)`.
- `StreamLogHandler` does not actually use the passed `source`.

Reply:
what are you proposing? to use `Logger.currentModule(filePath: file)` instead? I don't think it's public

Reply:
It's not, but you could use the same logic. It just takes the file path and does some string manipulation.

But since it's just being discarded by the logger, I'd just use an empty string or TuistSomething.

Reply:
If we have a way to simply disable the warning I would rather do that instead of copying logic from the library, or passing an empty string coupling it to the fact that such implementation doesn't use it